styles.css

```diff
@@ -0,0 +1,91 @@
+* {
+  box-sizing: border-box;
+  margin: 0;
+  padding: 0;
+  background-color: #07535b;
+}
+
+.header-toolbar {
+  display: flex;
+  flex-direction: row;
+  min-height: 4rem;
+  align-items: center;
+  padding: 15px;
+  color: white;
+  opacity: 1;
+  width: 100%;
+  justify-content: space-between;
+}
+
+.my-logo {
+  color: #fff;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 18px;
+  font-family: Roboto, sans-serif;
+  letter-spacing: 0.03em;
+  text-decoration: none;
+}
+
+.my-logo:hover {
+  text-decoration: none;
+}
+
+.my-logo:active {
+  text-decoration: none;
+}
+
+.my-logo:visited {
+  text-decoration: none;
+}
+
+.content-headline-section {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  text-align: center;
```

Comment:
this is not needed here as you already set it to center horizontally and vertically using flex properties and you added a text-align property each to its child elements.